src/internal/Common.h

```diff
@@ -25,6 +25,8 @@ typedef Range<const char*> StringPiece;
 
 namespace rsocket {
 
+constexpr int64_t kMaxRequestN = std::numeric_limits<int32_t>::max();
```

Comment:
`yarpl::credits::kNoFlowControl` in `yarpl/utils/credits.h`?  Any why is int32_t::max() being set into an int64_t?